index.html

```diff
@@ -1595,427 +1595,135 @@
-              return false;
-            }
-          } else {
-            console.log("Cannot handle " + task.taskType);
-            return false;
-          }
-        }
-
-        WorkerPool.prototype.Display = function() {
-          while(this.workingQueue.length &gt; 0) {
-            if (!!(this.workingQueue[0].displayTask)) {
-              var processWorker = this.workingQueue.shift();
-              postMessage(processWorker.displayTask);
-              processWorker.displayTask = null;
-              this.avaialableQueue.push(processWorker);
-            } else {
-              break;
-            }
-          }
-        }
-
-        WorkerPool.prototype.FindProcessWorker = function(worker) {
-          for (var i = 0; i &lt; this.workingQueue.length; ++i) {
-            if (this.workingQueue[i].worker == worker) {
-              return this.workingQueue[i];
-            }
-          }
-        }
-
-        /*
-         * ProcessWorker
-         */
-        function ProcessWorker(id, script, pool) {
-          this.workerID = id;
-          this.workerPool = pool
-          this.displayTask = null;
-          this.worker = new Worker(script);
-          this.worker.wraper = this;
-          this.worker.onmessage = function(event) {
-            var processWorker = workerPool.FindProcessWorker(this);
-            processWorker.displayTask = new DisplayTask(event.data.frameNum, event.data.bitmap);
-            workerPool.Display();
-            // workerPool.PrintInfo("in worker.onmessage()");
-          }
-        }
-
-        /*
-         * Task, ProcessTask, DisplayTask
-         */
-        function Task(type, frameNum) {
-          this.taskType = type;
-          this.frameNum = frameNum;
-        }
-
-        function ProcessTask(frameNum, bitmap) {
-          Task.call(this, ProcessTask.TASK_TYPE, frameNum);
-          this.bitmap = bitmap;
-        }
-        ProcessTask.prototype = Object.create(Task.prototype);
-        ProcessTask.prototype.constructor = ProcessTask;
-        ProcessTask.TASK_TYPE = "ProcessTask";
+      <section>
+        <h3>
+          Simple example
+        </h3>
+        <p>
+          This example demostrates how to hook a video worker to a
+          <a>MediaStreamTrack</a> as a <a>processing video worker</a> and the
+          script code of the video worker shows how to use the extended methods
+          of <a>ImageBitmap extensions</a>
+        </p>
+        <p>
+          The scenario is that captures real-time <a>MediaStream</a> from a
+          web camera, converts the captured color video frames into gray
+          scales and then shows both color and gray videos on the screen. Since
+          the captured color video is likes to be stored as YUV format natively,
+          this example asks the browser to map the video data into JavaScript
+          <a>ArrayBuffer</a> in any kind of YUV formats which might save the
```

Comment:
Add http://heycam.github.io/webidl/#idl-ArrayBuffer to dependencies after BufferSource.